src/IntegrationTests/resources/config/application_it.properties

```diff
@@ -21,3 +21,5 @@ evidence.welsh.hmctsImgVal="[userImage:welshhmcts.png]"
 
 cover.english=SSCS-cover-page.docx
 cover.welsh=TB-SCS-LET-WEL-00486.docx
+
+reinstatement_requests_feature_flag=false
```

Comment:
Why this feature flag is disable, are we not including integration test for this reinstatement

Reply:
Added IT